frontend/src/styles/_uswds-theme.scss

```diff
@@ -71,19 +76,35 @@ in the form $setting: value,
     )
   ),
   $theme-font-type-serif: "newsreader",
-  $theme-global-link-styles: true,
-  $theme-body-font-size: "md",
-  $theme-type-scale-3xs: 2,
+  // Type scale
+  $theme-type-scale-3xs: 1,
   $theme-type-scale-2xs: 3,
   $theme-type-scale-xs: 4,
   $theme-type-scale-sm: 5,
-  $theme-type-scale-md: 6,
+  $theme-type-scale-md: 7,
   $theme-type-scale-lg: 9,
   $theme-type-scale-xl: 12,
   $theme-type-scale-2xl: 15,
   $theme-type-scale-3xl: 17,
   $theme-font-role-heading: "sans",
   $theme-lead-font-family: "body",
+  // Headings
+  $theme-h1-font-size: "2xl",
+  $theme-h2-font-size: "xl",
+  $theme-h3-font-size: "lg",
+  $theme-h4-font-size: "md",
+  $theme-h5-font-size: "sm",
+  $theme-h6-font-size: "xs",
+  $theme-heading-line-height: 2,
+  // Paragraphs
+  $theme-text-measure-narrow: 4,
+  $theme-text-measure: 6,
+  $theme-text-measure-wide: none,
```

Comment:
- Adjust the type scale per convo w/ @crystabelrangel 
  - Use `sm` (`5` / 16px) as default font size
  - Make `md` noticeably different from `sm`
  - Set explicit sizes for headings (not `3xl` is not used by default, but reserved for larger custom classes as-needed)